docs/AboutUs.adoc

```diff
@@ -22,7 +22,8 @@ image::SakuraBlossom.png[width="150", align="left"]
 {empty}[http://github.com/SakuraBlossom[github]]
 
 Role: Team Lead +
-Responsibilities: Logic, Storage
+Responsibilities: Undo/Redo functionality, Eager Evaluation +
+Component(s) In Charge: Logic, Storage
```

Comment:
Can change to "CRUD for patients, doctors, appointment models."

Reply:
Add or Overwrite responsibilities?